### DISCUSSION TITLE
Passing variable

### DISCUSSION THREAD

**Discussion by beuzathor** [awcodes/filament-tiptap-editor#478] [Q&A]
I'm looking for a way to pass an additional object when using the method tiptap_converter()->asHTML. I haven't seen anything in the documentation and can't find anything here. Yet, it seems like something that would be quite useful. For example, something like {!! tiptap_converter()->asHTML->with($variable) !!}.

**Comment by awcodes** [comment]
There's nothing built in for that because the converter wouldn't know what to do with the $variable. 

You could extend the underlying class an add your own methods as needed. Look at `mergeTagsMap()` method to see an example.
